pipelines/smartseq2_single_sample/SmartSeq2SingleSample.wdl

```diff
@@ -55,14 +58,16 @@ workflow SmartSeq2SingleCell {
       fastq2 = fastq2,
       ref_name = hisat2_ref_name,
       sample_name = sample_name,
-      output_basename = quality_control_output_basename
+      output_basename = quality_control_output_basename,
```

Comment:
I like using trailing commas in lists because it cleans up diffs, but this is a style thing and I leave it up to you. 

(see how lines 79, 99, 107 aren't highlighted).